source/developer/fx-guidelines.rst

```diff
@@ -434,10 +434,47 @@ If what the users need to perform is obvious, we should make concious decisions
 ---------------------------
 
 |
+Feedback: Error Messages
+---------------------------
+
+On occasion, Mattermost users may encounter a problem that prevents them or the Mattermost system from successfully completing a task. The unexpected user behaviour or system response should be communicated to the user through an error message, and should follow the design principles of forgiving and obvious.
+ 
+Error messages should:
```

Comment:
There actually needs to be a line break before the first bullet or they dont render properly

![image](https://cloud.githubusercontent.com/assets/14333569/18769063/164f86b4-80e7-11e6-873b-eb97f42cc13c.png)